README.md

```diff
@@ -22,16 +22,33 @@ To activate it in your Jest environment you have to include it in your configura
 }
 ```
 
-### With Playwright test runner
+### With [Playwright test runner](https://playwright.dev/docs/test-intro)
 
-To activate with the Playwright test runner, use `expect.extend` to add the `expect-playwright` matchers.
+To activate with the Playwright test runner, use `expect.extend` in the config to add the `expect-playwright` matchers.
 
 ```js
-// folio.config.ts
+// playwright.config.ts
 import { expect } from "@playwright/test"
 import { matchers } from "expect-playwright"
 
 expect.extend(matchers)
+
+// ...
+```
+
+Inside the tests you can add the import to have TypeScript support.
+
+```js
+// foo.spec.ts
+import { test, expect } from "@playwright/test"
+import "expect-playwright"
```

Comment:
Wondering if we need this section given the TypeScript section at the bottom of the readme:

https://github.com/playwright-community/expect-playwright#typescript